lib/ansible/modules/cloud/misc/terraform.py

```diff
@@ -136,13 +136,13 @@
 module = None
 
 
-def preflight_validation(bin_path, project_path, variables_file=None, plan_file=None):
+def preflight_validation(bin_path, project_path, variables_args=None, plan_file=None):
```

Comment:
I think this needs to include both the vars args and vars file. Looks like excluding the vars_file in the preflight command was an oversight on my part when implementing this originally.

Reply:
Ah, good eye.